src/renderer/services/chain/fees/withdraw.ts

```diff
@@ -34,10 +34,10 @@ const withdrawFeeByChain$ = (chain: Chain): FeeLD => {
           )
         )
       )
-    case 'ETH':
-      return Rx.of(RD.failure(new Error('Deposit fee for ETH has not been implemented')))
     case 'THOR':
       return THOR.fees$.pipe(liveData.map(({ fast }) => fast))
+    default:
+      return Rx.of(RD.failure(Error(`Deposit fee for ${chain.toUpperCase} has not been implemented`)))
```

Comment:
Same here: Provide `failure` for all possible chains with proper error messages.